src/styles/application.css

```diff
@@ -10,6 +10,12 @@
   }
 }
 
+@media (min-width: 90rem) {
+  .sto-app {
+    width: 1220px; /* White space on the right on huge screen */
```

Comment:
We usually use `rem` as size unit in our styles. Could you use `75rem` (75*16=1200px) instead?